pvnet/data/wind_datamodule.py

```diff
@@ -1,9 +1,8 @@
 """ Data module for pytorch lightning """
```

Comment:
There are some things in this file that were commented out but for cleanliness should probably just be deleted